ui-v2/app/components/aria-menu/index.hbs

```diff
@@ -1,8 +1,10 @@
 {{yield
   (action 'change')
   (action 'keypress')
-  (concat 'component-aria-menu-trigger-' guid)
-  (concat 'component-aria-menu-menu-' guid)
-  (if expanded 'true' undefined)
   (action 'keypressClick')
+  (hash
+    labelledBy=(concat 'component-aria-menu-trigger-' guid)
+    controls=(concat 'component-aria-menu-menu-' guid)
+    expanded=(if expanded 'true' undefined)
+  )
```

Comment:
This moves the aria things around so they can be accessed via `aria.labelledBy` etc instead of `ariaLabelledBy`